content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+retornada e nenhum outro autorizador é consultado. Se todos os módulos de autorização não tiverem
+nenhuma opinião sobre requisição, então a requisição é negada. Uma negação retorna um
+código de status HTTP 403.
+
+## Revisão de atributos de sua requisição
+
+O Kubernetes revisa somente os seguintes atributos de uma requisição de API:
+
+ * **user** - O string de `user` fornecido durante a autenticação.
+ * **group** - A lista de nomes de grupos aos quais o usuário autenticado pertence.
+ * **extra** - Um mapa de chaves de string arbitrárias para valores de string, fornecido pela camada de autenticação.
+ * **API** - Indica se a solicitação é para um recurso de API.
+ * **Caminho da requisição** - Caminho para diversos endpoints sem recursos, como `/api` ou `/healthz`.
+ * **Verbo de requisição de API** - Verbos da API como `get`, `list`, `create`, `update`, `patch`, `watch`, `delete` e `deletecollection` que são utilizados para solicitações de recursos. Para determinar o verbo de requisição para um endpoint de recurso de API , consulte [Determine o verbo da requisição](/pt-br/docs/reference/access-authn-authz/authorization/#determine-the-request-verb).
+ * **Verbo de requisição HTTP** - Métodos HTTP em letras minúsculas como `get`, `post`, `put` e `delete` que são utilizados para requisições que não são de recursos.
+ * **Recurso** - O identificador ou nome do recurso que está sendo acessado (somente para requisições de recursos) -- Para requisições de recursos usando os verbos `get`, `update`, `patch` e `delete`, deve-se fornecer o nome do recurso.
+ * **Subrecurso** - O sub-recurso que está sendo acessado (somente para solicitações de recursos).
+ * **Namespace** - O namespace do objeto que está sendo acessado (somente para solicitações de recursos com namespace).
+ * **Grupo de API** - O {{< glossary_tooltip text="API Group" term_id="api-group" >}} sendo acessado (somente para requisições de recursos). Uma string vazia designa o _core_ [Grupo de API](/docs/reference/using-api/#api-groups).
+
+## Determine o verbo da requisição {#determine-the-request-verb}
+
+**Requisições de não-recursos**
+
+Requisições para endpoints diferentes de `/api/v1/...` ou `/apis/<group>/<version>/...`
+são considerados "requisições sem recursos" e usam o método HTTP em letras minúsculas da solicitação como o verbo.
+Por exemplo, uma solicitação `GET` para endpoints como `/api` ou `/healthz` usaria `get` como o verbo.
+
+**Requisições de recursos**
+Para determinar o verbo de requisição para um endpoint de API de recurso, revise o verbo HTTP
+utilizado e se a requisição atua ou não em um recurso individual ou em uma
+coleção de recursos:
+
+Verbo HTTP | Verbo de Requisição
+---------- |---------------
+POST       | create
+GET, HEAD  | get (para recursos individuais), list (para coleções, includindo o conteúdo do objeto inteiro), watch (para assistir um recurso individual ou coleção de recursos)
```

Comment:
Suggestion: use "observar" in lieu of "assistir".

Reply:
Fixed on b53e77805e